addons/mail/static/src/messaging/component/chatter/chatter.scss

```diff
@@ -9,32 +9,27 @@
     overflow: auto;
 }
 
-.o_FormRenderer_chatterContainer {
-    display: flex;
-    flex: 1 1 auto;
-    margin-left: auto;
-    margin-right: auto;
-    width: 100%;
+.o_Chatter_composer {
+    border-bottom: 1px solid;
 
-    @media (min-width: 576px) {
-        max-width: 540px;
-    }
-
-    @media (min-width: 768px) {
-        max-width: 720px;
-    }
-
-    @media (min-width: 992px) {
-        max-width: 960px;
+    &.o-bordered {
+        border-left: 1px solid;
+        border-right: 1px solid;
     }
+}
 
-    @media (min-width: 1200px) {
-        max-width: 1140px;
-    }
+/* Specificity needed to override generic style made in form_view.scss */
+.o_form_view .o_form_sheet_bg .o_FormRenderer_chatterContainer {
+    margin-left: auto;
+    margin-right: auto;
+}
```

Comment:
It's to center the chatter in the sheet as when put underneath it has a max-width and no fixed width (it has a fixed size when aside)

Reply:
Can you try again without the specificity? I get why we want it, but I don't see what other rules this is supposed to override.

Also, same remark as enterprise, this should be in form renderer widget scss file.

Reply:
It overrides this :
```css
@media (min-width: 1534px)
.o_form_view > .o_form_sheet_bg > :not(.o_form_sheet) {
    margin-left: -16px;
    margin-right: -16px;
}
```

Reply:
Could be useful to include path of style we want to be more specific.
In long-term we don't want any of these high-specificity rules, so when needed in the meantime, could help track which other rule is problematic.

Reply:
Wow... This is in enterprise right?

Do we ever have anything else that is directly inside `o_form_sheet_bg` and that is not a `o_form_sheet` except the chatter? Can't this just be removed since we now handle it correctly?

I think a big part `mail_enterprise.scss` could be deleted thanks to this PR.

Reply:
I don't know and it's difficult to check if anything else than a chatter could be in form_sheet_bg